lib/dialects/postgres/query-generator.js

```diff
@@ -375,7 +375,7 @@ var QueryGenerator = {
 
     var replacements = {
       table: this.quoteIdentifiers(tableName),
-      where: this.getWhereConditions(where),
+      where: this.getWhereConditions(where, null, model, options),
```

Comment:
Ah i see, thought we had killed all the syntax that simply uses where: 1.
Ok this is fine for now then.